src/main/scala/chiseltest/package.scala

```diff
@@ -422,8 +423,14 @@ package object chiseltest {
       inner
     }
 
-    def enumToString(record: EnumType): BigInt => String = {
-      def inner(bits: BigInt): String = "[unimplemented enum decode]"
+    def enumToString(tpe: EnumType): BigInt => String = {
+      def inner(bits: BigInt): String = {
+        val fullName = chisel3.internaltest.EnumHelpers.valueToName(tpe, bits).getOrElse("???")
+        // we only want to class and value name, not the package or enclosing class
+        val noPackage = fullName.split('.').takeRight(2).mkString(".")
+        val noEnclosingClass = noPackage.split('$').last
```

Comment:
Could you include an example of what fullName typically is as a comment to make the transforms on lines 430-1 clearer?

Reply:
I think that would be too noisy